content/c-sharp/concepts/math-functions/terms/asin/asin.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.Asin()'
+Description: 'Returns an angle in radians whose sine is a specified number.'
+Subjects: 
+  - 'Computer Science'
+Tags:
+  - 'Libraries'
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **Math.Asin()** function computes an **angle** in **radians** whose **sine** is a specified number. In Math, this is known as the **inverse sine** function.
```

Comment:
```suggestion
The **Math.Asin()** function computes an angle in radians whose sine is a specified number. In Math, this is known as the inverse sine function.
```

Reply:
@SSwiniarski have commited the changes as suggested, rgds